test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1006
+-- Description: PoliciesManager must allow all requested params in case "parameters" field is omitted
```

Comment:
Please update the description to actual one

Reply:
@GetmanetsIrina You can find the updates in the commit ac51975